src/vcpkg/commands.ci.cpp

```diff
@@ -678,6 +780,38 @@ namespace vcpkg::Commands::CI
             print2("\nTriplet: ", result.triplet, "\n");
             print2("Total elapsed time: ", GlobalState::timer.to_string(), "\n");
             result.summary.print();
+
+            if (baseline_iter != settings.end())
+            {
+                print2("\nREGRESSIONS:\n");
+                for (auto&& port_result : result.summary.results)
+                {
+                    switch (port_result.build_result.code)
+                    {
+                        case Build::BuildResult::BUILD_FAILED:
+                        case Build::BuildResult::POST_BUILD_CHECKS_FAILED:
+                        case Build::BuildResult::FILE_CONFLICTS:
+                            if (expected_failures.find(port_result.spec) == expected_failures.end())
+                            {
+                                std::cerr
```

Comment:
I think this should also use print2, and we certainly should avoiding new use of iostreams if at all possible.